one_off_script.py

```diff
@@ -5,9 +5,12 @@
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "django_scaffold.settings")
 django.setup()
 
+
 if __name__ == "__main__":
-    from one_off_scripts.backfill_daily_test_rollups import run_impl
-    from one_off_scripts.backfill_test_flag_bridges import backfill_test_flag_bridges
+    # from one_off_scripts.backfill_daily_test_rollups import run_impl
```

Comment:
Might be good to somehow control which scripts are run, instead of just commenting things out.
Otherwise, just removing the code for the ones that have been executed is also good.